packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.spec.js

```diff
@@ -85,9 +85,9 @@ describe('Component > FlipbookViewer', function () {
       const user = userEvent.setup({ delay: null })
 
       const { container, getByLabelText, getByTestId } = render(<DefaultStory />)
-      const imageSVGZoomLayer = getByTestId('zoom-layer')
+      const image = document.querySelector('image')
 
-      imageSVGZoomLayer.focus()
+      await user.click(image)
```

Comment:
Previously, you clicked on the event layer to focus the subject viewer. Now, you click directly on the subject image.